content/v2.0/get-started.md

```diff
@@ -65,6 +65,14 @@ Start InfluxDB by running the `influxd` daemon:
 ```bash
 influxd
 ```
+{{% warn %}}
+#### Influxd cannot be opened because the developer cannot be verified.
+With the latest release of MacOS Catalina some binaries cannot be run without first
+being authorized.  To allow these files to run open Spotlight search (⌘ + space) and
+type "Security".  Open the Security & Privacy panel and click "Allow Anyway".  
+Next time you run influxd you will be prompted to click "Open" and Influxd will run.
+This will be resolved soon.
```

Comment:
Just curious, what is the source of resolution? Catalina update, or something on the InfluxData side?

Reply:
There was a conversation internally, there are a couple of options but they all revolve around InfluxData signing the code to notarize it.  This will need to be done by InfluxData.

Reply:
Some copy updates:

- ~MacOS~ -> macOS (this is the official trademark)

```
#### Run InfluxDB on macOS Catalina
macOS Catalina requires downloaded binaries to be signed by registered Apple developers.
Currently, when you first attempt to run `influxd` or `influx`, macOS will prevent it from running.
To manually authorize the InfluxDB binaries:

1. Attempt to run the `influx` or `influxd` commands.
2. Open **System Preferences** and click **Security & Privacy**. 
3. Under the **General** tab, there is a message about `influxd` or `influx` being blocked.
   Click **Open Anyway**.
4. Repeat this process for both binaries.

We are in the process of updating our build process to ensure released binaries are signed by InfluxData.
```

Reply:
Fixed in the latest commit.